mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```
@@ -226,6 +226,11 @@
       "run": "after",
       "snippetPath": "create_index_on_job_execution_started_date.sql",
       "fromModuleVersion": "mod-source-record-manager-3.5.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "update_set_file_name_from_null_to_no_file_name.sql",
+      "fromModuleVersion": "mod-source-record-manager-3.6.0"
```

```suggestion
      "fromModuleVersion": "mod-source-record-manager-3.5.1"
```
It will be included in Nolana bugfix release